build/config/stage_envvars.yml

```diff
@@ -0,0 +1,27 @@
+stages:
+  - variables:
+      - description: Additional attributes for tagging
+        required: false
+        name: TF_VAR_attributes
+      - required: true
+        name: TF_VAR_name_company
+      - required: true
+        name: TF_VAR_name_component
+      - required: true
+        name: TF_VAR_name_project
+      - required: false
+        name: TF_VAR_tags
+      - description: " (Default: westeurope)"
+        required: false
+        name: TF_VAR_location
+      - description: " (Default: True)"
+        required: false
+        name: TF_VAR_registry_admin_enabled
+      - description: " (Default: standard)"
+        required: false
+        name: TF_VAR_registry_sku
+      - description: " (Default: stacks-docker-images)"
+        required: false
+        name: TF_VAR_resource_group_name
+
+    name: acr
```

Comment:
What's this?

Reply:
Do you mean this file?
If so then this is the configuration file for setting up a local environment for deploying the ACR for deploying images to from a branch.

Reply:
No, the acr bit on its own at the bottom haha